website/docs/golioth-exploration/01-golioth-intro/03-hardware-setup.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: connect-nRF7002-to-golioth
 description: |
-   Use a precompiled application to connect an nRF7002-DK to Golioth
+   Use a precompiled application to connect an nRF7002 DK to Golioth
```

Comment:
The Nordic branding doesn't have a `-` in the DK kit names, so I updated these to be consistent with what they are using on pages like: https://www.nordicsemi.com/Products/Development-hardware/nRF7002-DK